dev-python/nose/nose-1.3.7.recipe

```diff
@@ -0,0 +1,109 @@
+SUMMARY="Extends unittest to make testing easier"
+DESCRIPTION='nose extends the test loading and running features of unittest, \
+making it easier to write, find and run tests.
+
+By default, nose will run tests in files or directories under the current \
+working directory whose names include "test" or "Test" at a word boundary \
+(like "test_this" or "functional_test" or "TestClass" but not \
```

Comment:
A key contributor prefers to use double quotes, so I think we should do so :)
```diff
-DESCRIPTION='nose extends the test loading and running features of unittest, \
+DESCRIPTION="nose extends the test loading and running features of unittest, \
 making it easier to write, find and run tests.
 By default, nose will run tests in files or directories under the current \
 working directory whose names include "test" or "Test" at a word boundary \
-(like "test_this" or "functional_test" or "TestClass" but not \
+(like \"test_this\" or \"functional_test\" or \"TestClass\" but not \
-"libtest"). Test output is similar to that of unittest, but also includes \
+\"libtest\"). Test output is similar to that of unittest, but also includes \
 captured stdout output from failing tests, for easy print-style debugging.
 These features, and many more, are customizable through the use of plugins. \
 Plugins included with nose provide support for doctest, code coverage, and \
 profiling, flexible attribute-based test selection, output capture and more. \
 More information about writing plugins may be found on in the nose API \
 documentation, here:
-http://readthedocs.io/docs/nose/'
+http://readthedocs.io/docs/nose/"
```
BTW, since the above URL is already in `HOMEPAGE` maybe we can drop the last 3 lines:
```diff
-profiling, flexible attribute-based test selection, output capture and more. \
+profiling, flexible attribute-based test selection, output capture and more."
-More information about writing plugins may be found on in the nose API \
-documentation, here:
-http://readthedocs.io/docs/nose/'
```